bot/grab.py

```diff
@@ -82,7 +82,17 @@ def get_price(add_to_historical: bool = True) -> Dict:
     global historical_prices, hsp_head, session_struct
 
     initial_price = {}
-    prices = client.get_all_tickers()
+
+    # get all info on tickers from binance
+    # with retry on error reading
+    while True:
+        try:
+            prices = client.get_all_tickers()
+        except:
+            print(f"{txcolors.WARNING}Binance Problem Get All Tickers{txcolors.DEFAULT}")
+            time.sleep(0.2)
```

Comment:
would suggest a longer sleep time to prevent us from getting rate limited

Reply:
I hope the rate limiter of Binance work only on  good request... Because 1200 request by minutes get all tickers cost 2. You can do 600 call by minute .... 0.2 + time request you are under 600....